Package.swift

```diff
@@ -4,6 +4,7 @@ import PackageDescription
 let package = Package(
     name: "PostHog",
     platforms: [
+        // TODO: add  .visionOS(.v1), when SPM is >= 5.9
```

Comment:
Compatibility with older versions.
We support swift >= 5.3, not sure how is the adoption from there on, any ideas?

Reply:
Someone just this week wanted iOS 12 support so it's not safe to drop just because of that, not adding the SPM visionos target still works as it is so I'd rather keep it as it is with back compatibility.